src/main/java/ome/services/fulltext/FullTextBridge.java

```diff
@@ -23,8 +23,9 @@
 import ome.model.annotations.TextAnnotation;
 import ome.model.annotations.TermAnnotation;
 import ome.model.containers.Folder;
-import ome.model.core.Image;
-import ome.model.core.OriginalFile;
+import ome.model.core.*;
```

Comment:
Okay to do this?

Reply:
Yeah, I'm finding myself failing to be vigilant against intellij's defaults...

Reply:
I'd find the `.*` import much cleaner. Would it actually make any difference at the compiled byte code level? I'd guess not.

Reply:
You can adjust import class count in Intellij in the settings (Editor -> Code Style -> Java -- imports)

![Screenshot 2019-06-24 at 09 42 30](https://user-images.githubusercontent.com/3717090/60004618-e1297f80-9664-11e9-89e0-1719e0944f32.png)

Reply:
Agreed, but there's the requirement to remember to do that in every repository (I have each jar as a separate project)

Reply:
This is a matter of pref but we have avoided to have ``import .*``.
I personally prefer to be explicit

Reply:
Pushed a fix.